detox/src/devices/DeviceRegistry.js

```diff
@@ -44,10 +44,14 @@ class DeviceRegistry {
     });
   }
 
-  isDeviceBusy(deviceId) {
+  includes(deviceId) {
```

Comment:
usage of 'busy' here was wrong to begin with. this is a registry, it either includes or doesn't include a device.

Reply:
Okay, but semantically — what "inclusion" means? It means that a worker can no longer take this device, it is used somewhere else.